ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.index.tsx

```diff
@@ -60,8 +61,9 @@ import {
   TooltipProvider,
   TooltipTrigger,
 } from "@/components/ui/tooltip";
-import { useToast } from "@/components/ui/use-toast";
+import { useToast } from '@/components/ui/use-toast';
```

Comment:
Just as I see it: This will probably trivially conflict with https://github.com/eclipse-apoapsis/ort-server/pull/233.

Reply:
There will be conflicts, yes.

Reply:
We should simple rebase this once @mmurto's PR is merged and resolve the trivial conflicts along the way.

Reply:
What do you mean with "along the way"? Should I make a separate commit and fix the quotes in every file this PR touches?  And what's the rule exactly: is it single quotes everywhere, so a search-and-place?

Reply:
No need to search and replace just pnpm format.

Reply:
As for handling conflicts, I'd say the easiest way is to first accept your changes and then run format and add that to the same commit.

Reply:
I was actually talking about whether I need to do some formatting to this PR now - as I do not have `pnpm format`available until your PR is merged.

Reply:
Probably makes sense to just wait for it, or you could rebase over it and wait for it to be merged. Though if we review the PR as-is very strictly, this change on lines 63/64 should probably not be part of the current commits, but a part of separate formatting commit, as the change of `"` to `'` is not related to these.